metadata-ingestion/src/datahub/ingestion/source/redshift/lineage.py

```diff
@@ -107,7 +107,12 @@ def _get_sources_from_query(self, db_name: str, query: str) -> List[LineageDatas
         parser = LineageRunner(query)
 
         for table in parser.source_tables:
-            source_schema, source_table = str(table).split(".")
+            matches = len(str(table).split("."))
```

Comment:
Slightly cleaner imo to do:
```
split = str(table).split(".")
if len(split) == 3:
    ... = split
elif len(split) == 2:
    ... = split
```
Also should we add an else case that throws / handles an error?